client/store/reports/stats/resolvers.js

```diff
@@ -0,0 +1,26 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+const { apiFetch } = wp;
+import { dispatch } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { stringifyQuery } from 'lib/nav-utils';
+import { NAMESPACE } from 'store/constants';
+
+export default {
+	async getReportStats( state, endpoint, query ) {
+		try {
+			const report = await apiFetch( {
+				path: NAMESPACE + 'reports/' + endpoint + '/stats' + stringifyQuery( query ),
```

Comment:
Maybe we can check if `endpoint` starts with a `/`, and if it does, assume it's a full path & use it instead of `NAMESPACE + 'reports/' + endpoint + '/stats'`?

Reply:
What about having an array of known `v3` endpoints to check against? Then if the `endpoint` being passed in does not match any of those, we just use that string instead? Seems we won't be adding many new endpoints so there shouldn't be any overhead in maintaining the list.

Alternatively we could build the array dynamically in php off of registered endpoints and add to `wcsettings` though that seems overkill.